README.md

```diff
@@ -46,3 +46,23 @@ or with `npx`:
 ```sh
 $ npx add-dependencies /home/user/project/package.json moment@2.0.0 react@16.8 redux eslint --dev
 ```
+
+or via nodejs
+
+```js
+const npmAdd = require('add-dependencies');
+//OR
+// Note: to use import you must have `"type": "module"` in your projects package.json
+import npmAdd from 'add-dependencies';
+const dependencies = [
+    'package1',
+    'package2',
+    'package3',
+];
+const target = npmAdd.CONSTANTS.DEPENDENCIES;
```

Comment:
What is `npmAdd.CONSTANTS` here? Can't see any object like this over here.

Reply:
whoops missed this in the separation